.travis.yml

```diff
@@ -13,13 +16,23 @@ python:
   - "2.7"
 
 env:
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: "eNfbQI7Pkp3mjTFQgjrVVldmyE/Yn3+iCMVykhXppC85dJaCFMpjwPUfZa7p38tnnH7jhng2JL2tAXnJ4uOry0B3m9xYR7fNYXg/9uj1988FHvpDhfDFEKcMF8s70mr11HRijj6U8p6WDt/0ygD6aZ0LJzWzxhMFecFavOb6XOw="
+  matrix:
   - LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - TRANSIFEX="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="website_sale_vat_required"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="website_sale_vat_required"
```

Comment:
Don't we also need the INCLUDE lines?
@pedrobaeza do you know why EXCLUDE / INCLUDE are not mentioned at
https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml
?